tests/data/faultyDeviceReport.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This needs to be given a different extension (I suggest .jsonx which is used in Kettle tests) otherwise it will cause our linting task to fail